nd4j/nd4j-backends/nd4j-backend-impls/nd4j-cuda-preset/src/main/java/org/nd4j/presets/cuda/Nd4jCudaPresets.java

```diff
@@ -182,6 +183,7 @@ public void init(Logger logger, java.util.Properties properties, String encoding
     public void map(InfoMap infoMap) {
         //whether to include the SD_GCC_FUNCTRACE definition in the build. Not needed if we're not enabling the profiler.
         boolean funcTrace = System.getProperty("libnd4j.calltrace","OFF").equalsIgnoreCase("ON");
+        System.out.println("Functrace on: " + funcTrace);

```

Comment:
I guess this print is a left over from debugging?

Reply:
No I'm leaving that in to verify whether the presets detect functrace is on. The rest I've cleaned up.